project/ANT.c

```diff
@@ -75,6 +76,13 @@ static void ant_handle_transmit()
 	APP_ERROR_CHECK(err_code);
 }
 
+static void ant_handle_receieve(ant_evt_t * p_ant_evt) {
+	
+	if(p_ant_evt->message.ANT_MESSAGE_aucPayload[0] == RESET_IMPACT_COUNT) {
```

Comment:
nit: By looking at the surrounding functions, it appears the coding convention for this project would be to have opening brace on line 80, and leave the first line of code on line 81.
So like:

```
static void ant_handle_receieve(ant_evt_t * p_ant_evt)
{
    if(p_ant_evt->message.ANT_MESSAGE_aucPayload[0] == RESET_IMPACT_COUNT) {
    ...
```